lib/propolis/src/enlightenment/hyperv/mod.rs

```diff
@@ -358,16 +495,57 @@ impl MigrateMulti for HyperV {
             }
         }
 
+        // This enlightenment assumes that if a VM is exported and imported, the
+        // caller asking to import will configure the bhyve VM so that it has
+        // the same apparent guest frequency and offset the VM had when it was
+        // exported. Thus it's not necessary to change the reference TSC page's
+        // contents here. See the TSC module documentation for more details.
```

Comment:
is "TSC module" referring to the Hyper-V docs?

Reply:
Fixed in dce0275.